profiler/include/profile_batched_gemm_bias_gelu_gemm_bias_impl.hpp

```diff
@@ -0,0 +1,386 @@
+                }
+            }
+        }
+
+        auto ref_gemm1          = ReferenceGemm1Instance{};
+        auto ref_gemm1_invoker  = ref_gemm1.MakeInvoker();
+        auto ref_gemm1_argument = ref_gemm1.MakeArgument(acc0_g_m_n,
+                                                         b1_g_n_o,
+                                                         c1_g_m_o_host_result,
+                                                         a1_element_op,
+                                                         b1_element_op,
+                                                         c1_element_op);
+
+        ref_gemm1_invoker.Run(ref_gemm1_argument);
+
+        // bias
+        for(int b = 0; b < BatchCount; ++b)
+        {
+            for(int m = 0; m < M; ++m)
+            {
+                for(int o = 0; o < O; ++o)
+                {
+                    cde1_element_op(c1_g_m_o_host_result(b, m, o),
+                                    c1_g_m_o_host_result(b, m, o),
+                                    d1_g_m_o(b, m, o));
+                }
+            }
+        }
+    }
+
+    std::string best_op_name;
+    float best_ave_time   = 0;
+    float best_tflops     = 0;
+    float best_gb_per_sec = 0;
+
+    // profile device op instances
+    for(auto& op_ptr : op_ptrs)
+    {
+        auto argument_ptr = op_ptr->MakeArgumentPointer(
+            static_cast<A0DataType*>(a0_g_m_k_device_buf.GetDeviceBuffer()),
+            static_cast<B0DataType*>(b0_g_k_n_device_buf.GetDeviceBuffer()),
+            std::array<const void*, 1>{d0_g_m_n_device_buf.GetDeviceBuffer()},
+            static_cast<B1DataType*>(b1_g_n_o_device_buf.GetDeviceBuffer()),
+            static_cast<C1DataType*>(c1_g_m_o_device_buf.GetDeviceBuffer()),
+            std::array<const void*, 1>{d1_g_m_o_device_buf.GetDeviceBuffer()},
+            M,
+            N,
+            K,
+            O,
+            BatchCount,
+            StrideA0,
+            StrideB0,
+            std::array<ck::index_t, 1>{StrideD0},
+            StrideB1,
+            StrideC1,
+            std::array<ck::index_t, 1>{StrideD1},
+            BatchStrideA0,
+            BatchStrideB0,
+            std::array<ck::index_t, 1>{BatchStrideD0},
+            BatchStrideB1,
+            BatchStrideC1,
+            std::array<ck::index_t, 1>{BatchStrideD1},
+            a0_element_op,
+            b0_element_op,
+            cde0_element_op,
+            a1_element_op,
+            b1_element_op,
+            cde1_element_op);
+
+        auto invoker_ptr = op_ptr->MakeInvokerPointer();
+
+        if(op_ptr->IsSupportedArgument(argument_ptr.get()))
+        {
+            std::string op_name = op_ptr->GetTypeString();
+
+            float ave_time =
+                invoker_ptr->Run(argument_ptr.get(), StreamConfig{nullptr, time_kernel});
+
+            std::size_t flop      = (size_t(M) * N * K * 2 + size_t(M) * N * O * 2) * BatchCount;
```

Comment:
Extra flops by elementwise ops may be trivial so I am fine with evaluating only GEMM+GEMM's flops. However the number of bytes from the bias arrays d0's and d1's should play a significant factor in `num_btype` (not sure if it's typo of num_byte).

Reply:
done